_board/metro_m4_express.md

```diff
@@ -8,10 +8,33 @@ board_url: ""
 board_image: "/assets/images/boards/metro_m4_express.jpg"
 ---
 
-This board hasn't been fully documented yet. Please make a pull request adding more info to this file.
+The most powerful Metro at this time, the **Adafruit Metro M4** featuring the **Microchip ATSAMD51**. This Metro is like a bullet train, with it's 120MHz Cortex M4 with floating point support. Your code will zig and zag and zoom, and with a bunch of extra peripherals for support, this will for sure be your favorite new chipset.
 
-The description should be written to inform a CircuitPython user what makes the board unique and link to relevant info about it.
+To start off the ATSAMD51 journey it has a classic 'Arduino compatible' shape and pinout. This Metro is the same size as the others, and is compatible with many shields. It's got analog pins where you expect, and SPI/UART/I2C hardware support in the same spot as the Metro 328 and M0\. But! It's powered with an ATSAMD51J19:
 
+*   Cortex M4 core running at **120 MHz**
+*   [Floating point support with Cortex M4 DSP instructions](https://developer.arm.com/technologies/dsp/dsp-for-cortex-m)
+*   **512 KB** flash, **192 KB** RAM
+*   32-bit, 3.3V logic and power
+*   Dual 1 MSPS DAC (A0 and A1)
+*   Dual 1 MSPS ADC (8 analog pins)
+*   6 x hardware SERCOM (I2C, SPI or UART)
+*   22 x PWM outputs
+*   Stereo I2S input/output with MCK pin
+*   10-bit Parallel capture controller (for camera/video in)
+*   Built in crypto engines with AES (256 bit), true RNG, Pubkey controller
```

Comment:
Not sure I'd add these two since they aren't supported in CircuitPython. (Though the random number generator is.)

Reply:
removed!